src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -60,6 +60,14 @@ public class ScrollContentPresenter : ContentPresenter, IPresenter, IScrollable,
                 o => o.Viewport,
                 (o, v) => o.Viewport = v);
 
+        /// <summary>
+        /// Defines the <see cref="IsScrollChainingEnabled"/> property.
+        /// </summary>
+        public static readonly StyledProperty<bool> IsScrollChainingEnabledProperty =
+            AvaloniaProperty.Register<ScrollContentPresenter, bool>(
+                nameof(IsScrollChainingEnabled), 
+                defaultValue: true);
```

Comment:
Can you try this change? I think it should work as well.
Because in current implementation we basically have two property registrations with the same name.

```suggestion
            ScrollViewer.IsScrollChainingEnabledProperty.AddOwner<ScrollContentPresenter>();
```

Reply:
Everything else looks good

Reply:
Updated